.github/workflows/certora_4337.yml

```diff
@@ -23,12 +23,17 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
+      - name: Install pnpm
+        uses: pnpm/action-setup@v3
+        with:
+          version: 9
+
       - name: Install python
         uses: actions/setup-python@v4
         with: { python-version: 3.11 }
 
       - name: Install certora cli
-        run: pip install -Iv certora-cli==6.1.3
+        run: pip install -Iv certora-cli==7.3.0
```

Comment:
Context: Unrelated but opportunistic change (was tested with this during development, and there are no additional Certora related changes because of the version bump).